resources/migrations/001_update_migrations.yaml

```diff
@@ -5799,6 +5798,27 @@ databaseChangeLog:
         - customChange:
             class: "metabase.db.custom_migrations.DeleteScanFieldValuesTriggerForDBThatTurnItOff"
 
+  - changeSet:
+      id: v49.2024-05-07T10:00:00
+      author: qnkhuat
+      comment: Set revision.most_recent = true to latest revision and false to others. A redo of v48.00-008 for mysql
+      preConditions:
+        - onFail: MARK_RAN
+        - dbms:
+            type: mysql,mariadb
+      changes:
+        - sql:
+            dbms: mysql,mariadb
+            sql: >-
+              UPDATE revision r
+              JOIN (
+                  SELECT id, ROW_NUMBER() OVER (PARTITION BY model, model_id ORDER BY timestamp DESC, id DESC) AS row_num
```

Comment:
Why bother with timestamp at all, if we trust `id` for breaking ties why not rely on it being monotonic?

Reply:
Hmm, and in that case why not keep using the `r.id = subquery.max_id` pattern. I expect this will perform much better for large revision tables.

Reply:
> if we trust id for breaking ties why not rely on it being monotonic?

I'm not sure I trust it, tbh. With a lot of issues with MySQL, I don't have much faith in it.